native/acter/src/api/common.rs

```diff
@@ -442,19 +442,19 @@ impl LocationDesc {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct ReactionRecord {
     sender_id: OwnedUserId,
-    timestamp: MilliSecondsSinceUnixEpoch,
+    ts: MilliSecondsSinceUnixEpoch,
```

Comment:
any particular reason this expressive name has been replaced by a short less clear name? I see this also extended to the API and I am afraid this causes more confusion than it is helpful...

Reply:
`ReceiptRecord` has `ts` field too.
So I wanted to unify those fields in name.